data/summary/types/cgm_test.go

```diff
@@ -25,6 +25,8 @@ import (
 
 	"github.com/tidepool-org/platform/data/summary/types"
 
+	"gonum.org/v1/gonum/stat"
```

Comment:
nit: the order of these imports could be cleaned up

Reply:
fixed across all summary files.